src/pages/Transaction/TransactionCellList/index.tsx

```diff
@@ -54,9 +50,8 @@ export default ({
     [offset, cells.length],
   )
 
-  const handleAddrToggle = () => {
-    setIsAddrNew(!isAddrNew)
-  }
+  const [isDisplayDeprecatedAddresses, setDisplayDeprecatedAddresses] = useDisplayDeprecatedAddresses()
```

Comment:
How about naming them `[isDeprecatedAddressesDisplayed, setIsDeprecatedAddressesDisplayed]`

Reply:
Should the hook name and storage key also be changed?

Reply:
Sure, I think it's fine to update the temporarily stored value

Reply:
done